schemas/2015-01-01/Sendgrid.Email.json

```diff
@@ -0,0 +1,115 @@
+{
+  "id": "http://schema.management.azure.com/schemas/2014-04-01-preview/Sendgrid.json",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Sendgrid.Email",
+  "description": "Sendgrid Resource Types",
+  "resourceDefinitions": {
+    "accounts": {
+      "type": "object",
+      "properties": {
+        "name": {
```

Comment:
updated name definition
"name": {
          "anyOf": [
            {
              "type": "string",
              "minLength": 4,
              "maxLength": 50
            },
            {
              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
            }
          ],
          "description": "Name of the resource"
        },